packages/vega-typings/types/runtime/index.d.ts

```diff
@@ -25,10 +25,58 @@ export function timeFormatLocale(definition: object): void;
 // Parser
 export function parse(spec: Spec, config?: Config, opt?: { ast?: boolean }): Runtime;
 
+/** Types documented in vega-loader's README.md */
+type BaseLoaderOptions = {
+  /** Base URL prefix prepended to provided URI */
+  baseUrl: string;
+  /** Allows caller to explicitly set loading mode (local or network request). File mode only applies to server-side rendering. */
+  mode: 'file' | 'http';
+  /** Default protocol for protocol-relative URIs, defaults to HTTP */
+  defaultProtocol: string;
```

Comment:
Can we enumerate this?

Reply:
It looks like this can be file, http, a string that starts with `//`, or a bunch of other things: https://en.wikipedia.org/wiki/List_of_URI_schemes 

I'll include HTTP and file for a start, but leave a string at the end for those other cases.